seacatauth/openidconnect/handler/introspect.py

```diff
@@ -80,12 +80,12 @@ async def introspect(self, request):
 		client_id = params.get("client_id")
```

Comment:
Including the client_id and client_secret in the POST body is actually a form of OAuth2.0 Client Basic Authentication. Although admittedly I haven't included the client_secret parameter, since we only support _public_ clients, which do not need any secret (they can have one, but it adds little to no security since anyone can extract the secret).

Anyway, we will yet rethink the authentication here.

Reply:
Clarified on call